test/jest-setup.ts

```diff
@@ -1,10 +1,12 @@
+const matchers = require('jest-extended/all');
 import 'jest-extended';
 import 'jest-callslike';
 
 afterEach(() => {
 	jest.restoreAllMocks();
 	jest.clearAllMocks();
 });
+expect.extend(matchers);
```

Comment:
I realized that the @types of jest-extended is broken, not representing well the library anymore, so I had to fix that here